### ISSUE TITLE
[Accessibility: Resize text] New dialog is open at bottom of the page when zoom 200%

### ISSUE THREAD

**Issue by alinefm** [kimchi-project/wok#74]
When a new dialog is open to collect user inputs to a given action, it is being displayed at the bottom of the page.
The first impression is that the operation failed and no dialog was open. But after scrolling down you can see the dialog is here.
